modules/luci-base/htdocs/cgi-bin/luci

```diff
@@ -1,4 +1,10 @@
-#!/usr/bin/lua
+#!/bin/sh
```

Comment:
Out of my experience, but it looks a bit strange to me that the shebang is changed from lua to sh, when the majority of the code below is still lua.